source/Module/STIG/Functions.Checklist.ps1

```diff
@@ -392,7 +397,15 @@ function New-StigCheckList
                         if ($setting.InDesiredState -eq $true)
                         {
                             $status = $statusMap['NotAFinding']
-                            $comments = "Addressed by PowerStig MOF via $setting"
+                            if($null -ne $Verifier)
```

Comment:
A space is needed after the if, also, what do you think about using $PSBoundParameters.ContainsKey('Verifier') instead of $null -ne $Verifier?  This would be consistent with line 177.  Also, just a little parameter validation with ValidateNotNullOrEmpty, would ensure that there's a proper value coming in.